geop/src/main/scala/com/azavea/usace/programanalysis/geop/GeopServiceActor.scala

```diff
@@ -46,16 +48,17 @@ class GeopServiceActor(sc: SparkContext) extends Actor with HttpService {
       entity(as[CountArgs]) { args =>
         complete {
           future {
-            val multiPolygon = args.multiPolygon.reproject(LatLng, ConusAlbers)
-            val rasterLayers = ClippedLayers(args.rasters, multiPolygon, sc)
-            val rasterGroupedCount = RasterGroupedCount(rasterLayers, multiPolygon)
-
-            JsObject(
-              rasterGroupedCount
-                .map { case (keys, count) =>
-                  keys.mkString(",") -> JsNumber(count)
-                }
-            )
+            args.multiPolygons.map(m => {
+                val multiPolygon = m.reproject(LatLng, ConusAlbers)
```

Comment:
Consider indenting this block with two spaces to maintain consistency with the rest of the project.